### PR TITLE
Reworked paper to address more technical ADASS audience

### PR DESCRIPTION
**Pull request by frossie**

### CODE REVIEW

File: O3-2.tex [hunk 1]
```diff
@@ -55,82 +55,64 @@
...
+Following a competitive tender, we selected Google Cloud for this three-year activity to prepare our team for operational readiness and our community for interacting with Rubin data and services.
+The IDF is a full stand-alone facility (without dependance on on-premises computing) and is a real high-availability production environment for all services currently oriented to external users. These include the Rubin Science Platform as well as underlying services such as Qserv (our high performance database system) and Butler (data abstraction middleware).
 
+The architecture is depicted in Figure \ref{fig:IDFarch} and leverages the Google Kubernetes Engine (GKE) and Google Cloud Storage (GCS) to scale up and down as needed depending on demand.
+Kubernetes is our reference deployment platform both on-premises and in the Cloud as our services make heavy use of container orchestration patterns and consume containers produced by our build systems based on Jenkins and GitHub Actions,  produces Docker containers for deployment \citep[see e.g.,][]{2018SPIE10707E..09J}.
```
Comment (womullan) [inline O3-2.tex:74]:
```suggestion
Kubernetes is our reference deployment platform both on-premises and in the Cloud as our services make heavy use of container orchestration patterns and consume containers produced by our build systems based on Jenkins and GitHub Actions \citep[see e.g.,][]{2018SPIE10707E..09J}.
```

File: O3-2.tex [hunk 2]
```diff
@@ -55,82 +55,64 @@
...
+GitHub Issues were also used as the primary means for delegates to request technical assistance. All of the delegate pedagogical resources are publicly accessible.
 
-\section{Community engagement }
-The Rubin Community Engagement Team (CET) provided documentation and support to the scientists and students participating in DP0 as delegates.
-The term ``DP0 delegates'' was adopted to reflect DP0 participants’ important roles of representing the broad science community as learners, testers, and providers of feedback, and of sharing the benefits of their DP0 participation with their communities as teachers and colleagues.
+Some aspects of the Community Engagement model as applied to DP0 are designed to scale to thousands of users for the future Data Previews, such as the extensive documentation, the tutorials, and the use of the Community Forum.
+Other aspects, such as the frequent live virtual sessions, would not scale well to thousands of users and were instead specifically designed to ramp up skill levels from novice to intermediate in order to seed expertise with Rubin software in the community.
```
Comment (womullan) [inline O3-2.tex:95]: may not scale well - not sure this is the place for that discussion with CET ..

Comment (womullan) [inline O3-2.tex:95]:
```suggestion
Other aspects, such as the frequent live virtual sessions, may not scale well to thousands of users and were specifically designed to ramp up skill levels from novice to intermediate in order to seed expertise with Rubin software in the community.
```

**Review by womullan — CHANGES_REQUESTED** [review]

File: O3-2.tex (hunk 2 above)
Comment (frossie) [reply O3-2.tex:95]: I didn't write that text.

**Review by timj — APPROVED** [review]

**Pull request merged**